cpu/stm32f0/include/periph_cpu.h

```diff
@@ -101,6 +100,19 @@ typedef struct {
 } adc_conf_t;
 
 /**
+ * @brief   DAC line configuration data
+ */
+typedef struct {
+    gpio_t pin;             /**< pin connected to the line */
+    uint8_t chan;           /**< DAC device used for this line */
+} dac_conf_t;
+
+/**
+ * @brief   A CPU specific DAC line generator macro
+ */
+#define DAC_LINE_CONF(port, pin, chan)          { GPIO_PIN(port, pin), chan }
```

Comment:
I am not really a friend of this define, I think it does not really add much to the readability while making it one step harder to trace the code... But this might just be a personal opinion.